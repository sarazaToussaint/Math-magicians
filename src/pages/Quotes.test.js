import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quote from './Quote';

const QuoteTest = () => {
  <BrowserRouter>
    <Quote />
  </BrowserRouter>;
};

it('Renders correctly', () => {
  const TestQuotes = renderer.create(<QuoteTest />).toJSON();
  expect(TestQuotes).toMatchSnapshot();
});
