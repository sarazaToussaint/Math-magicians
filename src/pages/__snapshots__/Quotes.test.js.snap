// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `null`;
